### ISSUE TITLE
Switch Output does not enable controller setup

### ISSUE THREAD

**Issue by Invictaz** [jfedor2/hid-remapper#118]
The Nintendo Switch menu does not let the Switch output set by hid-remapper to be mapped.

The 8bitdo Adapter 2 does emulate a proper Pro Controller which can be remapped in the Switch menu

Maybe this is because there is some descriptor or report missing?
Same happens on https://github.com/juan518munoz/PicoSwitch-WirelessGamepadAdapter/issues/6

**Comment by jfedor2** [comment]
Yes, this is because Switch only allows mapping on the Pro Controller and we're emulating some weird Pokken thing.

**Comment by Invictaz** [comment]
@jfedor2 Can we emulate a Pro Controller?

**Comment by jfedor2** [comment]
> Can we emulate a Pro Controller?

I'm sure it's possible.
